packages/react-devtools-shared/src/hook.js

```diff
@@ -152,6 +166,151 @@ export function installHook(target: any): DevToolsHook | null {
     } catch (err) {}
   }
 
+  // Keep in sync with src/backend/utils.js
```

Comment:
nit: can we make these comments a little bolder?

```suggestion
  // NOTICE Keep in sync with src/backend/utils.js
```